### ISSUE TITLE
Please sync the test to intl-duration-format/pull PR181 

### ISSUE THREAD

**Issue by FrankYFTang** [tc39/test262#4050]
@ben-allen @anba 
Please fix  the following tests in test262 after the landing of the PR
https://github.com/tc39/proposal-intl-duration-format/pull/182
```
test262/intl402/DurationFormat/prototype/format/negative-duration-style-default-en
test262/intl402/DurationFormat/prototype/format/negative-durationstyle-long-en
test262/intl402/DurationFormat/prototype/format/negative-durationstyle-digital-en
test262/intl402/DurationFormat/prototype/format/negative-durationstyle-narrow-en
test262/intl402/DurationFormat/prototype/format/negative-duration-style-short-en
test262/intl402/DurationFormat/prototype/formatToParts/negative-duration-formatToParts-style-digital-en
test262/intl402/DurationFormat/prototype/formatToParts/negative-duration-formatToParts-style-long-en
test262/intl402/DurationFormat/prototype/formatToParts/negative-duration-formatToParts-style-narrow-en
test262/intl402/DurationFormat/prototype/formatToParts/negative-duration-formatToParts-style-short-en
test262/intl402/DurationFormat/prototype/formatToParts/negative-duration-formatToParts-style-default-en
```

These test call partitionDurationFormatPattern inside [test262](https://github.com/tc39/test262/tree/main)/[harness](https://github.com/tc39/test262/tree/main/harness)
/testIntl.js

But that file has not reflect the changes in PR182 yet.

**Comment by anba** [comment]
These tests are fixed by #3983.

**Comment by ben-allen** [comment]
Can we close this one?
